### PR TITLE
John conroy/update banner styles

### LINKED ISSUE #1077: Implement redesigned control banners
See #1063 for design. Make sure to update error message icon from "i" to "!".

### PR DESCRIPTION
**Pull request by john-conroy**
Fix #1077. The screenshots below are only meant to demonstrate the styling, ignore the text.

Warning
Screen Shot 2020-09-15 at 3 39 23 PM

Info
Screen Shot 2020-09-15 at 3 39 38 PM

Error
Screen Shot 2020-09-15 at 3 39 50 PM

Success
Screen Shot 2020-09-15 at 3 40 10 PM

### CODE REVIEW

**Review by tsliaw — APPROVED** [review]
Can you make sure all text colors in the banners are #000000?

**Comment by john-conroy** [comment]
Screen Shot 2020-09-16 at 1 38 10 PM

@tsliaw is there a specific font size you want for the alert messages?

**Comment by tsliaw** [comment]
Yes, body2 font

**Comment by john-conroy** [comment]
@mccalluc not sure what's going on with travis here. I am getting different errors on each re-run with certain checks failing while others pass (no consistent outcome). The changes to `package-lock.json` are just from an `npm audit` and are unrelated.

**Comment by mccalluc** [comment]
Huh. Let me look into it...

**Comment by mccalluc** [comment]
Looking at the errors:
```
18:17:44]  Downloading Cypress     [started]

No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
```
```
curl: (35) gnutls_handshake() failed: The TLS connection was non-properly terminated.

Version '12.16.3' not found - try `nvm ls-remote` to browse available versions.

The command "nvm install 12.16.3" failed and exited with 3 during .
```
```
The command '/bin/sh -c npm ci' returned a non-zero code: 1
```
They all might be network problems... nothing on https://www.traviscistatus.com/ ... but network problems wouldn't necessarily show up?

Going to try restarting the `Travis CI - Branch` build....

**Comment by mccalluc** [comment]
There doesn't seem to be a way to un-cancel a build, but a rerun fixed the other one that was failing. Just Travis flakiness. 🤷

File: context/app/static/js/shared-styles/alerts/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  />
+);
+
+const StyledAlert = styled(OutlinedAlert)`
+  :not(svg) {
+    color: ${(props) => props.theme.palette.text.primary};
+  }
+`;
+
+export { StyledAlert as Alert };
```
Comment (mccalluc) [inline context/app/static/js/shared-styles/alerts/index.js:28]: I'm fine with this, but do you think we should have fewer `Styled*` exports generally? Don't care much one way or the other.

**Review by mccalluc — APPROVED** [review]
One question, looks fine.

**Pull request merged**